### ISSUE TITLE
Add link explaining the Safari workaround

### ISSUE THREAD

**Issue by kaycebasques** [PolymerLabs/actor-boilerplate#13]
https://github.com/PolymerLabs/actor-boilerplate/blob/master/src/bootstrap.ts#L25

Got an issue or PR that explains why this is needed?

**Comment by surma** [comment]
it says right above. Or are you looking for something more in-depth? It's necessary to keep the worker alive. Safari seems to kill workers that don't receive messages.

**Comment by kaycebasques** [comment]
Right, I saw that. I was looking for more info. Just curious about the behavior.

**Comment by surma** [comment]
There really isn’t much more info than that :D We were just wondering why our worker suddenly stopped polling idb and it seems that Safari assumes that if a worker doesn’t receive or send messages after ~3 seconds, it’s superfluous and kills it. 

Sending messages, even if there’s no message handler in the worker, seems to prevent Safari from killing it.

**Comment by kaycebasques** [comment]
Welp, the good news is that we're creating the documentation now with this thread!
